### PR TITLE
Updating API Activity to handle Container Activity Logs 

### PR DESCRIPTION
**Pull request by pladam**
Updating API Activity to handle Container Activity Logs:

1. Adding NEW verbosity field to metadatajson object and dictionary.json 
2. Adding adding data fields to request.json and response.json objects to handle the kubernetes response and request objects
3. Adding namespace field to resource_details.json object to handle api namespaces
4. Adding group object to api.json object to handle api groups

### CODE REVIEW

**Review by rroupski — CHANGES_REQUESTED** [review]

File: dictionary.json [hunk 1]
```diff
@@ -3314,6 +3314,11 @@
       "caption": "Rate Limit",
       "description": "The rate limit for a rate-based rule.",
       "type": "integer_t"
+    },
+    "verbosity": {
```
[the remark this answers is not in the document]
Comment (pladam) [reply dictionary.json]: This has been updated.

File: dictionary.json [hunk 2]
```diff
@@ -907,6 +907,12 @@
       "description": "The information describing an instance of a container. A container is a prepackaged, portable system image that runs isolated on an existing system using a container runtime like containerd.",
       "type": "container"
     },
+    "containers": {
```
Comment (rroupski) [inline dictionary.json:910]: There are too many places where the containers attribute is used. The API Activity class has 4 of them. See the arrows on the picture below pointing to the Container object. We should simplify it.

image

File: profiles/container.json [hunk 3]
```diff
@@ -4,11 +4,11 @@
   "caption": "Container",
   "name": "container",
   "attributes": {
-    "container": {
+    "containers": {
```
Comment (rroupski) [inline profiles/container.json:7]: Missing group properties, such as primary, context, etc.

File: profiles/container.json [hunk 4]
```diff
@@ -4,11 +4,11 @@
...
       "requirement": "recommended"
     },
     "namespace_pid": {
```
Comment (rroupski) [inline profiles/container.json:10]: Missing group properties, such as primary, context, etc.

File: profiles/container.json (hunk 3 above)
Comment (rroupski) [inline profiles/container.json:7]: Remaining the `container` as `containers` is not a good idea. It breaks the existing definitions. For example, now the `process` object has a `containers` attribute, which is wrong.

 Also, this change is not a backward compatible, and we should bot make any breaking changes in the v1.1.

**Review by rroupski — CHANGES_REQUESTED** [review]

File: profiles/container.json (hunk 3 above)
Comment (pladam) [reply profiles/container.json:7]: how can we account for events which have multiple containers then?

File: dictionary.json (hunk 2 above)
Comment (pladam) [reply dictionary.json:910]: for the sake of the log im attempting to model in particular, there are an array of containers in both the request and response of the api. any suggestions here to simplify?

Comment (rroupski) [reply dictionary.json:910]: Sure, can you send me some sample data?

**Pull request closed without merging**